src/main/java/dev/jbang/cli/Jdk.java

```diff
@@ -23,10 +25,15 @@
 	public Integer install(
 			@CommandLine.Option(names = { "--force",
 					"-f" }, description = "Force installation even when already installed") boolean force,
-			@CommandLine.Parameters(paramLabel = "version", index = "0", description = "The version to install", arity = "1") int version)
+			@CommandLine.Parameters(paramLabel = "version", index = "0", description = "The version to install", arity = "1") int version,
+			@CommandLine.Parameters(paramLabel = "existingJdkPath", index = "1", description = "Pre installed JDK path", arity = "0..1") String path)
 			throws IOException {
 		if (force || !JdkManager.isInstalledJdk(version)) {
-			JdkManager.downloadAndInstallJdk(version);
+			if (StringUtils.isNotBlank(path)) {
```

Comment:
Prefer the usage of https://github.com/jbangdev/jbang/blob/238ef90b0f65aba896b241b06bc8cde796364382/src/main/java/dev/jbang/util/Util.java#L1339_L1349 over the use of external libraries. (This might change in the future but at least that way it would be easy to find all occurrences)